contracts/strategies/allocation/no-application/README.md

```diff
@@ -0,0 +1,38 @@
+Spec: No application with NFT contract
+---------------------------------
+
+## Overview 
+Some builders have flagged that they want the ability create pools that don't require applications, where anyone who has taken some action is automatically an eligible recipient in their pool. This is a strategy stub to test that format. 
```

Comment:
Should we pick one of these "actions" and show an example of it?

Reply:
there is some interesting discussion about this use case over at https://twitter.com/owocki/status/1675555641254096896